upb/def.c

```diff
@@ -2693,36 +2691,60 @@ static void resolve_fielddef(symtab_addctx *ctx, const char *prefix,
       // No resolution necessary.
       break;
   }
+}
+
+static void resolve_extension(
+    symtab_addctx *ctx, const char *prefix, upb_fielddef *f,
+    const google_protobuf_FieldDescriptorProto *field_proto) {
+  if (!google_protobuf_FieldDescriptorProto_has_extendee(field_proto)) {
+    symtab_errf(ctx, "extension for field '%s' had no extendee",
```

Comment:
I am guessing this contains a longjmp or something, but would lightly prefer a return so I don't have to worry about it

Reply:
Unfortunately that would require a separate `return` statement. C with `-pedantic` doesn't allow `return f()` in a function returning `void`.

Another option is to make all the functions in this file return a `bool`, even though that return value will never actually used.

Another option is to drop `-pedantic`, though that could compromise the portability.

None of these options seem great. :/